### ISSUE TITLE
Differences from tinfoil

### ISSUE THREAD

**Issue by miiichael** [gsurrel/MessengerBypasser#6]
Hi, I'm trying to use your 2.0 APK but am not having much luck.
- many links (eg. login button!) want to open in Chrome or Tinfoil instead of Bypasser itself
- the swipe-from-right menus do not work

**Comment by gsurrel** [comment]
Hi,

I've indeed not finished to work on the app (too many things to do) and it currently works well enough for my use: even if a link wants to open elsewhere, the request is done (login, start new message thread…). 

It's right it's nothing near "release quality" but I did that as a temp quick fix: I'm not sure what strategy I want to follow as I do not want to become a maintainer of Tinfoil and I don't find any good way to use the behavior of the 1.x branch while keeping Tinfoil as-is. The best would probably to fork Tinfoil and just edit the user agent.

I'll keep the issue open in the meantime.
